src/index.html

```diff
@@ -10,8 +10,150 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+
+      <fieldset class="container">
+        <legend>Personal information</legend>
+        <label class="form-field">
+            Surname:
+          <input type="text" name="Surname">
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input type="text" name="Name">
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input 
+            type="number"
+            name="Age"
```

Comment:
names still not fixed, use camelCase 
```suggestion
            name="age"
```
[checklist p.4 ](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

Reply:
fix it all over the code